pkg/controller/bundlec/bundle_sync_task.go

```diff
@@ -130,38 +136,38 @@ func (st *bundleSyncTask) processNormal() (retriableError bool, e error) {
 		}
 		st.processedResources[resourceName] = &resInfo
 	}
-	err := st.findObjectsToDelete()
+	external, retriable, err := st.findObjectsToDelete()
 	if err != nil {
-		return false, err
+		return external, retriable, err
 	}
 	if st.isBundleReady() {
 		// Delete objects which were removed from the bundle
 		retriable, err := st.deleteRemovedResources()
 		if err != nil {
-			return retriable, err
+			return false, retriable, err
 		}
 	}
 
-	return false, nil
+	return false, false, nil
 }
 
 // Process the bundle marked with DeletionTimestamp
 // TODO: remove this method after https://github.com/kubernetes/kubernetes/issues/59850 is fixed
-func (st *bundleSyncTask) processDeleted() (retriableError bool, e error) {
+func (st *bundleSyncTask) processDeleted() (externalError bool, retriableError bool, e error) {
```

Comment:
Would be great to remove named return values everywhere for style consistency. No need to do it in this PR, perhaps later.